R/compat-plyr.R

```diff
@@ -0,0 +1,334 @@
+# Adds missing elements to a vector from a default vector
+defaults <- function(x, y) c(x, y[setdiff(names(y), names(x))])
```

Comment:
Can we export `defaults()` also? In my experience, it's the plyr function most likely to be used by extension developers, since it's used to adjust default aesthetics in geoms.

Reply:
Sure